eq-author/src/components/ContentPicker/GroupContentPicker.js

```diff
@@ -20,6 +20,31 @@ const ActionButtons = styled(ButtonGroup)`
   flex: 0 0 auto;
 `;
 
+const generateDefaultState = (selectedDestination, config) => {
+  if (!selectedDestination) {
+    return {
+      openLevel: null,
+      selectedItem: null,
+    };
+  }
+  Object.keys(selectedDestination).forEach(key => {
```

Comment:
Instead of clearing things out of the object. Could we instead search through the possible locations?
so 35 becomes something like: `destinationType = selectedDestination.page || ...`

Reply:
Also this is very routing specific for a `GroupContentPicker`